### PR TITLE
Update suitcase-intro.rst

### PR DESCRIPTION
**Pull request by Shubhamr837**
Changed Instructions to download.
Added instructions to delete from CLI.
Added information about new tabs.

### CODE REVIEW

File: odkx-src/suitcase-intro.rst [hunk 1]
```diff
@@ -57,26 +59,40 @@ ODK-X :guilabel:`Cloud Endpoint Address`, the :guilabel:`App ID`, and your :guil
...
 
   .. note:: 
-    By default :guilabel:`App ID = default`, check the second paragraph of :ref:`ODK-X Data Management Applications <build-app-data-mgt-apps>` to see more details.
+    By default :guilabel:`App ID = default`,to use your own App ID uncheck the option to use default App ID, check the second paragraph of :ref:`ODK-X Data Management Applications <build-app-data-mgt-apps>` to see more details.
 
-There are two tabs at the top of the graphical interface:
-  - :guilabel:`Download`, to download existing data from the server
-  - :guilabel:`Upload`, to upload new data to the server. The :guilabel:`Upload` tab also includes an option to :guilabel:`Reset` the server.
+Suitcase remembers the last username and server url entered. You can check the :guilabel:`save password` option to keep yourself logged in when you open the application again. To log out of the application you can use the logout button in a menu on the top left of the Application.
+
```
Comment (elmps2018) [inline odkx-src/suitcase-intro.rst:65]: Please add a screenshot of the logout button I couldn't see it in the screenshots. Also application can be lower case

File: odkx-src/suitcase-intro.rst [hunk 2]
```diff
@@ -57,26 +59,40 @@ ODK-X :guilabel:`Cloud Endpoint Address`, the :guilabel:`App ID`, and your :guil
...
+  - :guilabel:`Download`, to download existing data from the server
+  - :guilabel:`Upload`, to upload new data to the server.
+  - :guilabel:`Update`, to update existing data on the server.
+  - :guilabel:`Clear`, to delete tables from the server. This tab also has the option to reset the entire server.
 
   .. _suitcase-gui-download:
 
 Downloading from the server
-  When downloading, you will need to specify the *table_id*.
+  When downloading, you will need to select the Table ID to download from the dropdown. The dropdown will contain all the Table IDs present in the cloud endpoint server.
```
Comment (elmps2018) [inline odkx-src/suitcase-intro.rst:80]: suggest using table_id here and throughout (as in line 82)

File: odkx-src/suitcase-intro.rst [hunk 3]
```diff
@@ -57,26 +59,40 @@ ODK-X :guilabel:`Cloud Endpoint Address`, the :guilabel:`App ID`, and your :guil
...
+  If you don’t see a Table ID in the dropdown which is  present in the cloud endpoint server click the :guilabel:`Refresh Tables List Metadata` button.
+  After selecting a table_id in the dropdown you can click on the :guilabel:`Add` button to add the Table ID in the list of selected Table IDs for download.
+  This way you can add multiple table ids for download at once. You can use the :guilabel:`Add All` button to add all the Tables at once.
```
Comment (elmps2018) [inline odkx-src/suitcase-intro.rst:83]: tables lower case table_ids with underscore.

File: odkx-src/suitcase-intro.rst [hunk 4]
```diff
@@ -107,14 +123,40 @@ Uploading to the server
...
   Then modify the :guilabel:`Upload` field to that file path by clicking on the :guilabel:`...` button, and then press :guilabel:`Upload`.
 
+  Screenshot of upload tab:
+
+  .. image:: /img/suitcase-intro/upload_tab.png
+   :width: 600
+
   .. note::
 
       Suitcase GUI supports only Uploading of files and not Updating or Modifying, check out :ref:`Suitcase CLI  <suitcase-cli-update>`
```
Comment (elmps2018) [inline odkx-src/suitcase-intro.rst:133]: This seems like it is now incorrect?

File: odkx-src/suitcase-intro.rst [hunk 5]
```diff
@@ -107,14 +123,40 @@ Uploading to the server
...
 
   .. _suitcase-gui-reset:
   
 Resetting the server
-  The :guilabel:`Reset` button can be found under the :guilabel:`Upload` tab. Clicking it will reset the the server without any warning or confirmation.
+  The :guilabel:`Reset` button can be found under the :guilabel:`Clear` tab. Clicking it will reset the the server after a warning and a confirmation.
```
Comment (elmps2018) [inline odkx-src/suitcase-intro.rst:138]: "Clicking it" is a bit unclear folks might worry it is the clear tab. Suggest "Clicking 'reset'

Comment (elmps2018) [reply odkx-src/suitcase-intro.rst:138]: Also should reference clear png here. Make sure all pngs are embedded.

File: odkx-src/suitcase-intro.rst [hunk 6]
```diff
@@ -107,14 +123,40 @@ Uploading to the server
...
+   :width: 600
+
+  .. _suitcase-gui-delete:
+  
+Deleting from the server
+  The :guilabel:`Delete` button can be found under the :guilabel:`Clear` tab. Select the Table ID to delete from the dropdown and click on delete. Selected Table ID will be deleted from the server after a warning and a confirmation.
+
+  Screenshot of Clear Tab:
+
+  .. image:: /img/suitcase-intro/upload_tab.png
```
Comment (elmps2018) [inline odkx-src/suitcase-intro.rst:158]: I think this should be clear not upload

File: odkx-src/suitcase-intro.rst [hunk 7]
```diff
@@ -163,6 +205,7 @@ CLI commands
     -updateLogPath <arg>        Specify a custom path to create update log
                                 file. Default directory is ./Update
     -upload                     Upload one file or all files in directory
+    -delete                     Delete a table in server
```
Comment (elmps2018) [inline odkx-src/suitcase-intro.rst:208]: does this need the table name, e.g. -delete <table_id>

File: odkx-src/suitcase-intro.rst [hunk 8]
```diff
@@ -113,34 +113,35 @@ Downloading from the server
...
-      Suitcase GUI supports only Uploading of files and not Updating or Modifying, check out :ref:`Suitcase CLI  <suitcase-cli-update>`
-
   .. _suitcase-gui-reset:
-  
+
 Resetting the server
-  The :guilabel:`Reset` button can be found under the :guilabel:`Clear` tab. Clicking it will reset the the server after a warning and a confirmation.
+  The :guilabel:`Reset` button can be found under the :guilabel:`Clear` tab. Clicking :guilabel:`Reset` will reset the the server after a warning and a confirmation.
+
+  Screenshot of Clear Tab:
```
Comment (elmps2018) [inline odkx-src/suitcase-intro.rst:136]: I think this should have the same :guilabel:'Clear' formatting. Tab should be lower case.

**Review by elmps2018 — COMMENTED** [review]
Thanks for the good revisions! One additional minor comment.

**Comment by elmps2018** [comment]
Looks like one of the checks failed on CircleCI -- you should be able to look at the details to figure out the issue.

**Comment by Shubhamr837** [comment]
@elmps2018 I have Updated the formatting for download, update and upload also. Please suggest if anything else can be done to make it better.

**Review by elmps2018 — COMMENTED** [review]
These look good to me, @EstherPhebe may have some further follow up. 
Also assuming we should hold off on merging these until the new app goes out.

**Comment by EstherPhebe** [comment]
@elmps2018 Looks good to me. Awesome work @Shubhamr837.

I agree on holding off merging.

**Review by EstherPhebe — APPROVED** [review]

**Review by wbrunette — APPROVED** [review]

**Pull request merged**